### ISSUE TITLE
PoE support for `cm4-carrier-net`

### ISSUE THREAD

**Issue by StefanUlbrich** [jkiv/cm4-carriers#4]
Hi! Thanks for the interesting project.

Have you considered adding PoE support to the `cm4-carrier-net`? This would render it in a very neat, minimalistic node for a kubernetes cluster.

**Comment by Ll3ma** [comment]
i was just about to ask for that and a usb port for things like NAS applications

**Comment by svanimisetti** [comment]
+1 on this request.

**Comment by jkiv** [comment]
@StefanUlbrich Yup! I've considered this. The latest version still needs to be built and tested for correctness. PoE will likely be the next step.
